src/renderer/App.css

```diff
@@ -4,8 +4,86 @@
 /** GitHub Primer Design System */
 @import "@primer/css/dist/primer.css";
 
-/** Tailwind CSS Configuration */
-@config '../../tailwind.config.ts';
```

Comment:
It it possible to make the tailwind change separately, to reduce complexity and risk in this PR?